tests/phpunit/tests/Hooks/TestFrontend.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace WPML\PB\Elementor\Hooks;
+
+/**
+ * @group hooks
+ * @group frontend
+ * @group wpmlcore-7279
+ */
+class TestFrontend extends \OTGS_TestCase {
+
+	/**
+	 * @test
+	 */
+	public function itShouldAddHooks() {
+		$subject = new Frontend();
+		\WP_Mock::expectActionAdded( 'elementor_pro/search_form/after_input', [ $subject, 'addLanguageFormField' ] );
+		$subject->add_hooks();
+	}
+
+	/**
+	 * @test
+	 */
+	public function itShouldAddLanguageFormField() {
+		$subject = new Frontend();
+	   \WP_Mock::expectAction( 'wpml_add_language_form_field' );
+	   $subject->add_hooks();
```

Comment:
@GeorgeBotsev, this is not correct (and tests are failing). You need to call the `addLanguageFormField` method:

```
$subject->addLanguageFormField();
```

Sorry, the example in my previous comment was wrong...

Reply:
And please check the indentation.